src/components/color-picker/swatches.ts

```diff
@@ -23,20 +23,52 @@ export const colors = [
 
 export const brightnesses = ['lighter', 'light', 'default', 'dark', 'darker'];
 
+export interface Swatch {
+    name: string;
+    value: string;
+    disabled?: boolean;
+}
+
 /**
+ * Returns the CSS variable name holding the RGB triplet for the color & brightness.
  *
- * @param color
- * @param brightness
+ * @param color the base color identifier (e.g. "red", "blue")
+ * @param brightness the brightness variant (e.g. "light", "default")
+ * @returns CSS variable name in the form --color-{color}-{brightness}
  */
 export function getColorName(color: string, brightness: string): string {
     return `--color-${color}-${brightness}`;
 }
 
 /**
- *
+ * Swatch value: inline CSS color value in the required format: rgb(var(--color-*-*))
  * @param color
  * @param brightness
  */
-export function getCssColor(color: string, brightness: string): string {
+export function getSwatchValue(color: string, brightness: string): string {
     return `rgb(var(${getColorName(color, brightness)}))`;
 }
+
+/**
+ * Swatch name: human readable label like "red default"
+ * @param color
+ * @param brightness
+ */
+export function getSwatchName(color: string, brightness: string): string {
```

Comment:
Why would we export it? It is used only in this file

Reply:
this is our default "palette". So we need to give it to somehow give it to the palette component to render it as swatches.